### DISCUSSION TITLE
Would forgejo be eligable?

### DISCUSSION THREAD

**Discussion by Andre601** [simple-icons/simple-icons#8269] [General]
I'm curious to know if the Open-source project [forgejo](https://codeberg.org/forgejo/forgejo/) would be eligable for being added to Simple icons.
I ask because it isn't purely a website, but also not a GitHub-hosted project, so it means criterias may differ and depend on your judgement (As mentioned in the contributing.md file).

I've linked the project repository above. And I assume you would probs use the 5k stars requirement here (Given that it is hosted on a repo that allows stars)? Tho I would consider this unfair as codeberg is a relatively unknown page compared to GitHub or even GitLab, so maybe other conditions/requirements should be considered?

I also want to mention, that I'm not asking on behalf of forgejo themself. I'm just curious to know if this project had a chance of getting their icon added to simple icons (If they are obviously okay with that).

**Comment by ripzerotwo** [comment]
If im being honest no.

**Reply by ripzerotwo** [reply]
Though this would be a question for the project managers. We have quite a open issues that dont meet requirements which make it an interesting question to ask. I personaly have been ignoring any issue that doesnt meet basic reqs so yeah.

**Comment by adamrusted** [comment]
So we are able to make use of multiple metrics when considering a brand. For example, if another open-source project is hosted on Codeberg, but we've historically added that project because of a _website_ rank - we can try and extrapolate roughly what website rank / stars might equate to. Another option would be to prove that it is equally popular, or moreso, to similar packages/applications using monthly downloads, or another definitive metric in that regard.

On the Codeberg stars, and website ranking alone - this would be well out of scope for the project at the moment.
